binding/file_request_type_binder.go

```diff
@@ -0,0 +1,65 @@
+package binding
+
+import (
+	"net/http"
+	"reflect"
+
+	"github.com/monoculum/formam"
+)
+
+// FileRequestTypeBinder is in charge of binding File request types.
+type FileRequestTypeBinder struct{}
+
+// RegisterTo register this RequestTypeBinder to the passed request binder
+// on the File content types (multipart/form-data).
+func (ht FileRequestTypeBinder) RegisterTo(binder *RequestBinder) {
+	binder.Register("multipart/form-data", ht.binder(binder.formDecoder))
```

Comment:
`binder.formDecoder` is not-exported, so how would a third party write this same binder?

Reply:
I did rewrite it. Take a look at it and LMK if it makes sense.